components/search/package.json

```diff
@@ -20,15 +20,15 @@
   "peerDependencies": {
     "@spectrum-css/clearbutton": "^1.0.0",
     "@spectrum-css/icon": "^3.0.0",
-    "@spectrum-css/textfield": "^3.0.0 || ^4.0.0",
-    "@spectrum-css/vars": ">=9.0.0"
+    "@spectrum-css/textfield": "^4.0.0",
+    "@spectrum-css/tokens": ">=8.0.0"
```

Comment:
```suggestion
    "@spectrum-css/textfield": ">=4",
    "@spectrum-css/tokens": ">=8 <=9"
```

Reply:
I wanted to confirm that `@spectrum-css/tokens` change here. Did you mean for that to be `"@spectrum-css/tokens": ">=9"`? I've been meaning to ask about how we're generally doing these now. I think I've been seeing most of the devDependencies using the `>=` now for tokens (and vars) in recent changes. Are we using `>=` now generally for the components as well now? (`@spectrum-css/textfield` change suggested here, and I saw a few others in a recent rebase).